nep-X1.mediawiki

```diff
@@ -27,17 +27,19 @@ Neo N3 has the native <code>ContractManagement</code> contract that is responsib
 {
   "name": "destroy",
   "safe": false,
-  "parameters": [],
+  "parameters": not restricted,
   "returntype": "Void"
 }
 </pre>
 This method MUST invoke the <code>destroy</code> method of <code>ContractManagement</code> contract when the contract is destroyed.
 
 <code>destroy</code> method can have multi parameters for any specific purpose since it's an operation only for contract owner.
 
-This function SHOULD check whether the <code>signer</code> address equals the <code>owner</code> hash of contract to avoid security risks. This function SHOULD use the SYSCALL <code>Neo.Runtime.CheckWitness</code> to verify the <code>signer</code>. Details has been explained in [NEP-30](https://github.com/superboyiii/proposals/blob/upgrade-standard/nep-30.mediawiki).
+<code>destroy</code> method SHOULD do checkwitness for contract <code>owner</code> to avoid security risks. This function SHOULD use the SYSCALL <code>Neo.Runtime.CheckWitness</code> to verify the <code>signer</code>. Details has been explained in [NEP-30](https://github.com/superboyiii/proposals/blob/upgrade-standard/nep-30.mediawiki).
```

Comment:
The link is to some non-canonical repository and NEP-30 is `verify()`, it's _not_ related to this check, contract can have no `verify()` and still have a properly functioning `destroy()`.